security/appsec/intro/what-else-do-you-need-to-know.md

```diff
@@ -22,6 +22,10 @@ Before diving into this skill, you should make sure you have a strong foundation
 
 > 💡 As with all the skills in the app, if you want to access more advanced or specific content at any time, **feel free to use the *Skill Map* tab** to jump to the workouts most relevant for you.
 
+> 💬 Why are you interested in learning Security?
```

Comment:
same comment here, maybe we can say "learn about Security" ?

cc @nem035 @Loopiezlol 

```suggestion
> 💬 Why are you interested in learning about Security?
```